csrankings-9.csv

```diff
@@ -1316,6 +1317,7 @@ Yang Hua,Queen's University Belfast,https://www.qub.ac.uk/schools/eeecs/Connect/
 Yang Liu 0003,Nanyang Technological University,http://www.ntu.edu.sg/home/yangliu,NOSCHOLARPAGE
 Yang Liu 0004,University of Texas at Dallas,http://www.hlt.utdallas.edu/~yangl,NOSCHOLARPAGE
 Yang Liu 0005,Tsinghua University,http://nlp.csai.tsinghua.edu.cn/~ly,NOSCHOLARPAGE
+Yang Liu 0007,Hong Kong Baptist University,http://www.comp.hkbu.edu.hk/~csygliu/,NOSCHOLARPAGE

```

Comment:
Listed as "research" professor, but only tenure-track faculty are currently eligible for inclusion.

Reply:
The personal webpage is outdated. He is now an assistant professor. See https://www.comp.hkbu.edu.hk/v1/?page=faculty

Reply:
I have removed this entry.